src/mixins/common-mixin.ts

```diff
@@ -78,7 +86,15 @@ export class CommonMixin extends Vue {
       case CorpTypeCd.BC_CCC: return EntityTypes.CC
       case CorpTypeCd.BC_COMPANY: return EntityTypes.CR
       case CorpTypeCd.BC_ULC_COMPANY: return EntityTypes.UL
+      // case CorpTypeCd.CONTINUE_IN: return ''           // CorpTypeCd.CONTINUE_IN = 'C'
       case CorpTypeCd.COOP: return EntityTypes.CP
+      case CorpTypeCd.EXTRA_PRO_A: return EntityTypes.XCR // CorpTypeCd.EXTRA_PRO_A = 'A'
+      // case CorpTypeCd.LIMITED_CO: return ''            // CorpTypeCd.LIMITED_CO = 'LLC'
+      case CorpTypeCd.SOCIETY: return EntityTypes.SO
```

Comment:
Couple questions here. Any change I do I'll be sure to reflect it in the code block (`entityTypeToCorpType`) above as well.
1. Are `CorpTypeCd.CONTINUE_IN` and `CorpTypeCd.LIMITED_CO` legal codes that we need to be handling in NameRequest? If yes what would be their respective `EntityTypes` codes? If not I will remove them from the search query string in [`business-lookup-services.ts`](https://github.com/bcgov/namerequest/blob/main/src/services/business-lookup-services.ts#L40)
2. What exactly is `CorpTypeCd.EXTRA_PRO_A`? I took a guess at what it's related `EntityTypes` codes would be but I may be incorrect.

Reply:
Does this help?

https://github.com/bcgov/namex/blob/068d6b6f3d7eef914b93a412424ed747ddd5846f/api/namex/services/lookup/name_request_filing_actions.py#L51

Reply:
Yes it did! As for `CorpTypeCd.CONTINUE_IN`, I don't think we use it anywhere so I will be removing it.

Reply:
1. EntityTypes.LLC is already mapped to CorpTypeCd.LIMITED_CO (in enums/entity-types.ts).

2. I think you're right that CorpTypeCd.CONTINUE_IN (C) is not used... per this: [name request types.txt](https://github.com/bcgov/namerequest/files/12686743/name.request.types.txt)

Reply:
For 1. I think I have the proper entity codes now for LIMITED_CO and EXTRA_PRO_A. For LIMITED_CO I can't use LLC since it is not one of the [entity types in namex](https://github.com/bcgov/namex/blob/main/api/namex/constants/__init__.py#L81), but namex uses RLC for Xpro Limited Liability Companies.

For 2. I'll await Mihai's reply in the team's thread.

Reply:
What did you use for LIMITED_CO?

Did you confirm that EXTRA_PRO_A is an Xpro Limited Liability Company (RLC)?

Reply:
For `LIMITED_CO` I used `EntityTypes.RLC`. In the raw_nr_to_action mapping ([L62-L65](https://github.com/bcgov/namex/blob/068d6b6f3d7eef914b93a412424ed747ddd5846f/api/namex/services/lookup/name_request_filing_actions.py#L62-L65)) you provided, `LIMITED_CO` (LLC) maps to 4 different NR type codes (LC, CLC, RLC and AL, depending on the action), but only RLC (Xpro Limited Liability Company) is used in namex `EntityTypes`. It seems like we allow it to be used for restorations and name changes in [request-action-mapping.ts](https://github.com/bcgov/namerequest/blob/main/src/list-data/request-action-mapping.ts#L62).

`EXTRA_PRO_A` also maps to 4 different codes in raw_nr_to_action ([L101-L104](https://github.com/bcgov/namex/blob/068d6b6f3d7eef914b93a412424ed747ddd5846f/api/namex/services/lookup/name_request_filing_actions.py#L101-L104)): UA, XUL, XCUL, and XRUL, but only XUL (Xpro Unlimited Liability Company) is used in namex `EntityTypes`.

Reply:
I agree with you. OK by me to resolve this conversation.